go/vt/sqlparser/truncate_query.go

```diff
@@ -17,17 +17,28 @@ limitations under the License.
 package sqlparser
 
 import (
-	"flag"
+	"github.com/spf13/pflag"
+
+	"vitess.io/vitess/go/vt/servenv"
 )
 
 var (
 	// TruncateUILen truncate queries in debug UIs to the given length. 0 means unlimited.
-	TruncateUILen = flag.Int("sql-max-length-ui", 512, "truncate queries in debug UIs to the given length (default 512)")
+	TruncateUILen int
 
 	// TruncateErrLen truncate queries in error logs to the given length. 0 means unlimited.
-	TruncateErrLen = flag.Int("sql-max-length-errors", 0, "truncate queries in error logs to the given length (default unlimited)")
+	TruncateErrLen int
 )
 
+func registerQueryTruncationFlags(fs *pflag.FlagSet) {
+	fs.IntVar(&TruncateUILen, "sql-max-length-ui", 512, "truncate queries in debug UIs to the given length (default 512)")
```

Comment:
if you define the variables with their default, you can do:

```go
fs.IntVar(&TruncateUILen, "...", TruncateUILen, "...")
// ditto ErrLen
```

Reply:
Fixed